package/MDAnalysis/topology/PDBParser.py

```diff
@@ -159,6 +166,7 @@ class PDBParser(TopologyReaderBase):
      - resids
      - resnames
      - segids
+     - elements
 
```

Comment:
In addition to this, could you also add a `.. versionchanged:: 1.0.0` section in the docstring? Saying that valid elements are now read.